casts/cards/src/UICollectionCard/UICollectionCard.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import { StyleSheet } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+import { ColorVariants, makeStyles, Theme, useTheme } from '@tonlabs/uikit.themes';
+import { TouchableOpacity } from '@tonlabs/uikit.controls';
+import { UILayoutConstant, UISkeleton } from '@tonlabs/uikit.layout';
+import type { UICollectionCardProps } from './types';
+import { QuickView } from './QuickView';
+import { Title } from './Title';
+import { Badge } from './Badge';
+import { UIConstant } from '../constants';
+
+const linearGradientStart = 'rgba(16, 19, 21, 0.8)';
+const linearGradientEnd = 'rgba(16, 19, 21, 0.2)';
+
+export const UICollectionCard: React.FC<UICollectionCardProps> = ({
```

Comment:
I like the naming! :)

Though please make it with a named `function` as it works better for development (please consider that we must use it for all components instead of anonymous functions)